src/display/ai/ai_view.py

```
@@ -0,0 +1,27 @@
+from src.display.themes.dark_theme import SYMBOLS as sym
```

Ruff (0.8.2)

`src.display.themes.dark_theme.SYMBOLS` imported but unused

Remove unused import: `src.display.themes.dark_theme.SYMBOLS`

(F401)